build/yamls/base/crds.yml

```diff
@@ -871,8 +871,64 @@ spec:
                         type: string
                       lastTransitionTime:
                         type: string
+    - name: v1alpha3
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                childGroups:
+                  x-kubernetes-preserve-unknown-fields: true
+                podSelector:
+                  x-kubernetes-preserve-unknown-fields: true
```

Comment:
Could you make the verification more strict like https://github.com/vmware-tanzu/antrea/pull/2125?

Reply:
Done